models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -25,39 +26,16 @@ WITH pool_labels AS (
             decimals,
             AVG(price) AS price
         FROM {{ source('prices', 'usd') }}
-        WHERE blockchain = "arbitrum"
+        WHERE blockchain = 'arbitrum'
         GROUP BY 1, 2, 3
     ),
 
-    dex_prices_1 AS (
```

Comment:
Wondering if it makes sense to translate the `dex_prices` code and just comment out the lines where it is used. Easier for updating in the future.

Reply:
agreed. will fix